chrome/browser/extensions/api/cookies/cookies_api.cc

```diff
@@ -207,25 +225,25 @@ CookiesGetFunction::CookiesGetFunction() {
 CookiesGetFunction::~CookiesGetFunction() {
 }
 
+std::unique_ptr<api::cookies::Get::Params> CookiesGetFunction::GetParsedArgs() {
+  return Get::Params::Create(*args_);
+}
+
+bool CookiesGetFunction::SetStoreBrowserContext() {
+  return extensions::SetStoreBrowserContext(this, parsed_args_, 
+    store_browser_context_);
+}
+
 bool CookiesGetFunction::RunAsync() {
-  parsed_args_ = Get::Params::Create(*args_);
+  parsed_args_ = GetParsedArgs();
   EXTENSION_FUNCTION_VALIDATE(parsed_args_.get());
```

Comment:
Why is this changed, while the body of GetParsedArgs() and the line being replaced looks identical.